This example is throwing a lot of errors

So after a straight clone, `npm install` and replacement of the `account_id` running `wrangler publish` I get:

```
ERROR in ./src/index.js 1:13-32
Module not found: Error: Can't resolve 'slug.txt' in '/Users/tylervanderhoeven/Desktop/modules-webpack-commonjs/src'
```

After fixing that error to use a relative import I get:

```
ERROR in ./src/slug.txt 1:6
Module parse failed: Unexpected token (1:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> Hello from Wrangler!
 @ ./src/index.js 1:13-34
```

If I just remove that import entirely and just hard code the string I get:

```
✨  Build completed successfully!
Error: Something went wrong! Status: 400 Bad Request, Details {
  "result": null,
  "success": false,
  "errors": [
    {
      "code": 10021,
      "message": "Uncaught TypeError: Main module name is not present in bundle.\n  at line 0\n"
    }
  ],
  "messages": []
}
```

Either a lot of assumptions are being made or this is really out of date with the way es-modules actually work. Would love a fix to this so that it actually works. Thanks!

--- a/src/index.js
+++ b/src/index.js
@@ -1,4 +1,4 @@
-const slug = require("slug.txt")
+const slug = require("./slug.txt")
 
 exports.handlers = {
   async fetch(request, env) {

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -10,6 +10,12 @@ module.exports = {
     libraryTarget: 'commonjs2',
     sourceMapFilename: 'index.js.map',
   },
+  externals: {
+    // The externals section is how you tell webpack "I want this module to be resolved at runtime,
+    // instead of at build time". This is required for all modules that you upload separately from
+    // the bundle
+    './slug.txt': 'commonjs2 ./slug.txt'
+  },
   optimization: {
     minimize: true,
   },

--- a/wrangler.toml
+++ b/wrangler.toml
@@ -11,3 +11,6 @@ command = "npm install && npm run build"
 [build.upload]
 format = "modules"
 main = "./shim.mjs"
+[[build.upload.rules]]
+type = "Text"
+globs = ["**/*.txt"]

